src/node_contextify.cc

```diff
@@ -583,6 +595,11 @@ class ContextifyScript : public BaseObject {
     else if (produce_cached_data)
       compile_options = ScriptCompiler::kProduceCodeCache;
 
+    Context::Scope scope(
+        maybe_context.IsEmpty() ?
+            env->context() :
+            maybe_context.ToLocalChecked());
```

Comment:
Could be shortened to `Context::Scope scope(maybe_context.FromMaybe(env->context()));`.